pandas/_libs/tslibs/offsets.pyx

```diff
@@ -743,6 +747,25 @@ cdef class Tick(SingleConstructorOffset):
                 "Tick offset with `normalize=True` are not allowed."
             )
 
+    # FIXME: Without making this cpdef, we get AttributeError when calling
+    #  from __mul__
+    cpdef Tick _next_higher_resolution(Tick self):
```

Comment:
wouldn't this be better as a module level routine?

Reply:
i dont think so.  it uses `self.n`, so really makes sense as a method